src/AdminAcceptanceTests.Steps/Utils/UITest.cs

```diff
@@ -12,16 +11,18 @@ public sealed class UITest
         internal IWebDriver Driver;
         internal PageActionCollection Pages;
         internal readonly string Url;
+        private readonly Settings _settings;
+        internal readonly User AdminUser;
```

Comment:
(Ignoring `_settings`) convert to auto-properties if they need to be `internal`?